test-serviced-deps.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+set -e
+set -x
+
+cd $CC_RPM_DIR
+sudo yum -y localinstall --disablerepo=\* --enablerepo=zenoss-mirror ${CC_RPM_FILENAME}
```

Comment:
Maybe add some kind of echo message before to explain that the script is about to install  `${CC_RPM_FILENAME}` using just the dependencies in the yum mirror to verify that mirror is complete